test-browser.js

```diff
@@ -2,12 +2,33 @@
 // $ browserify test-browser.js | pbcopy
 'use strict';
 const publicIp = require('./browser');
+const test = require('tape');
 
-(async () => {
-	console.log('IP:', await publicIp.v4());
-	console.log('IP:', await publicIp.v4({
+let v6Address = null;
+
+test('v6', async t => {
+	try {
+		const ip1 = await publicIp.v6();
+		v6Address = await publicIp.v6();
+
+		t.equal(ip1, v6Address);
+		t.end();
+	} catch (error) {
+		if (error === 'Couldn\'t find your IP') {
+			v6Address = null;
```

Comment:
I'm not sure what the point of setting this to null is.

Shouldn't this case fail the test?

Reply:
Sorry, my bad, it was the leftover code from the previous `v6or4` function test change. I should have taken a second look